recipes/reindent/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "Reindent" %}
+{% set version = "0.1.1" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "d61226a0d0109aef3495033b42e8a0258c6bef1c014d9f26e31c62f071352b82" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: 'python setup.py install '
+
+requirements:
+  build:
+    - python 2.*
+  run:
+    - python 2.*
+
+test:
+  imports:
+    - reindent
```

Comment:
Can you pleas put in a few more tests? Import tests would be good, but actual tests would be good too (if there are any).